app/persistence/status/status.go

```diff
@@ -1,5 +1,63 @@
 package status
 
+import (
+	"errors"
+	"github.com/hashgraph/hedera-sdk-go"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"strconv"
+	"time"
+)
+
 // This table will contain information for latest status of the application
 type Status struct {
+	Name      string
+	AccountID string
+	Code      string
+	Value     string
+}
+
+type StatusRepository struct {
+	dbClient *gorm.DB
+}
+
+func NewStatusRepository(dbClient *gorm.DB) *StatusRepository {
+	return &StatusRepository{
+		dbClient: dbClient,
+	}
+}
+
+func (s StatusRepository) GetLastFetchedTimestamp(accountID hedera.AccountID) string {
+	lastFetchedStatus := &Status{}
+	failure := s.dbClient.
+		Table("statuses").
```

Comment:
I don't think that you need to define the `table` at all. You can use the `Status`